rmf_demo_panel/task_list/clinic_tasks.json

```diff
@@ -0,0 +1,6 @@
+[
+{ "task_type":"Loop", "start_time":0, "description": {"num_loops":5, "start_name":"L1_left_negative_air_pressure", "finish_name":"L2_left_nurse_center"}},
+{ "task_type":"Loop", "start_time":1, "description": {"num_loops":5, "start_name":"L1_left_nurse_center", "finish_name":"L2_south_couter"}},
```

Comment:
I made a typo here previously: should rename `L2_south_couter` as `L2_south_counter`

Reply:
ok will update!